src/test/java/core/basesyntax/service/RegistrationServiceTest.java

```diff
@@ -0,0 +1,99 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceTest {
+    private static final String ACCEPTABLE_LOGIN = "SimpleLogin";
+    private static final String EMPTY_STRING = "";
+    private static final String TWO_CHARACTER_STRING = "Hi";
+    private static final String FIVE_CHARACTER_STRING = "Hello";
+    private static final int NEGATIV_AGE = -20;
+    private static final int ZERO_AGE = 0;
+    private static final int POSITIV_UNACCEPTABLE_AGE = 17;
+    private RegistrationService registrationService;
+    private List<User> users = new ArrayList<>();
+    private User user1;
+    private User user2;
+    private User user3;
```

Comment:
Create an inside test case where you need this data. 
```suggestion
    private RegistrationService registrationService;

```